engine/link/mystery_gift.asm

```diff
@@ -814,14 +814,14 @@ ReceiveInfraredLEDOff:
 	ldh [rIF], a
 	halt
 	ldh a, [c]
-	bit rRP_RECEIVING, a
+	bit RPF_WRITE_HI, a
 	jr nz, .no_recv_loop
 	or a
 	ret
 
 SendInfraredLEDOn:
 ; Holds the IR LED on for d-1 interrupts.
-	ld a, rRP_ENABLE_READ_MASK | (1 << rRP_LED_ON)
+	ld a, RPF_ENREAD | (1 << RPF_WRITE_LO)
```

Comment:
It does not make sense to shift *by* a bitmask (i.e. `1 << REGB_THING` = ok, `REGF_THING` = ok, `1 << REGF_THING` = fishy).
```suggestion
	ld a, RPF_ENREAD | RPF_WRITE_HI
```